src/nvim/lua/executor.c

```diff
@@ -916,12 +916,22 @@ int nlua_in_fast_event(lua_State *lstate)
   return 1;
 }
 
+static bool viml_func_is_deferred_safe(const char_u *name)
```

Comment:
This name is confusing because it returns `fast` but that's the opposite of "deferred".

Let's avoid needless entropy by using the same jargon everywhere by default (i.e. unless there's a clear reason to diverge). Consistent with `nlua_in_fast_event`, I would suggest a name like `viml_func_is_fast`.